src/components/CurrentTask/CurrentTask.js

```diff
@@ -1,14 +1,29 @@
 import useStore from '../../hooks/useStore';
+import Button from '../Button/Button';
 
 import StyledCurrentTask from './StyledCurrentTask';
 
 export default function CurrentTask() {
 	const tasks = useStore(state => state.tasks);
+	const deleteTask = useStore(state => state.deleteTask);
 
-	return (
-		<StyledCurrentTask>
-			<p>{tasks[0].name}</p>
-			<p>{tasks[0].time} min</p>
-		</StyledCurrentTask>
-	);
+	function handleClick() {
+		deleteTask(tasks[0].id);
+	}
+
+	if (tasks.length) {
+		return (
+			<StyledCurrentTask>
+				<p>{tasks[0].name}</p>
+				<p>{tasks[0].time} min</p>
+				<Button onClick={handleClick}>done</Button>
```

Comment:
You can pass here directly your `deleteTask` function instead of using the `handleClick` function.

```suggestion
				<Button onClick={()=> deleteTask(tasks[0].id)}>done</Button>
```